public/main.css

```diff
@@ -256,6 +259,8 @@ main {
 }
 
 .key {
+  transition: background 0.5s;
+  transition-delay: 1s;
```

Comment:
I think it's nicer to delay the keyboard update until the tiles have updated

Reply:
That's a good change.

Reply:
Should it be longer?